proto/osmosis/gamm/v1beta1/pool.proto

```diff
@@ -35,6 +35,8 @@ message PoolParams {
     (gogoproto.moretags) = "yaml:\"exit_fee\"",
     (gogoproto.nullable) = false
   ];
+  string swapFeeGovernor = 4
+      [ (gogoproto.moretags) = "yaml:\"swapFeeGovernor\"" ];
```

Comment:
`yaml:\"swapFeeGovernor\"` => `yaml:\"swap_fee_governor\"` ?

Reply:
Yeah that's better